### PR TITLE
Fix reference preview by downgrading banana-i18n

### PR DESCRIPTION
**Pull request by stephanebisson**
Phabricator Link: https://phabricator.wikimedia.org/T263109

### Problem Statement

banana-i18n was update from 1.2.1 to 1.3.1 (even though it was not needed) in #271 to fix npm audit warnings. That version turns out to error out on the reference dialog title message. Will follow up with banana-i18n separately. 

### Solution

Downgrade banana-i18n from 1.3.1 to 1.2.1

### CODE REVIEW

**Review by hueitan — APPROVED** [review]
Confirm the fix.

**Comment by jpita** [comment]
How can we avoid this in the future (besides having more automated tests for all the user flows possible)?

**Comment by jpita** [comment]
can anyone run `npm test` on their machine to see the result of the tests?
@stephanebisson the version of firefox on circleci is 66 while the latest version is 80, is there any way to update circleci?

**Comment by jpita** [comment]
the test passes on my local without the `--silent` flag but fails with the `--silent` flag with firefox 80.
on circle ci fails even without the flag on firefox 66.
I created an issue on the cypress repo so I'll put this test on skip https://github.com/cypress-io/cypress/issues/8610

File: cypress/integration/references-popup.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  beforeEach(() => {
+    cy.navigateToHomePage()
+  })
+
+  it.skip('check preview for references opens', () => {
+    goToCatArticle()
+    cy.downArrow()
+    cy.rightArrow()
+    cy.enter()
+    cy.get('.ref-title').should('be.visible').should('contain.text', 'Reference [10]')
```
Comment (hueitan) [inline cypress/integration/references-popup.js:17]: Hey @jpita, mine was opening one article preview in my local

image

**Pull request merged**